tests/vppoutputencode.cpp

```diff
@@ -186,3 +186,19 @@ bool VppOutputEncode::output(const SharedPtr<VideoFrame>& frame)
     return true;
 
 }
+
+bool VppOutputEncode::flush()
```

Comment:
could we move this  VppOutputEncode::~VppOutputEncode::()?
you may need check m_encoder and m_output  before use it.